app/client/ui/WidgetTitle.ts

```diff
@@ -210,12 +210,15 @@ function buildWidgetRenamePopup(ctrl: IOpenController, vs: ViewSectionRec, optio
   let widgetInput: HTMLInputElement|undefined;
   let descInput: HTMLTextAreaElement | undefined;
   return cssRenamePopup(
+    // Create a FocusLayer to keep focus in this popup while it's active, and prevent keyboard
+    // shortcuts from being seen by the view underneath.
+    elem => { FocusLayer.create(ctrl, { defaultFocusElem: elem, pauseMousetrap: false }); },
     dom.onDispose(onClose),
     dom.autoDispose(commandGroup),
     testId('popup'),
     dom.cls(menuCssClass),
     dom.maybe(!options.tableNameHidden, () => [
-      cssLabel(t("DATA TABLE NAME")),
+      cssLabel(t("Data table name")),
```

Comment:
Why the change? I think this breaks existing translations.

Reply:
oooh you're right, I forgot that .. --'
I had changed it because I think it's cleaner to say it's uppercase in the css because it's UI decision, but better not to break existing translations